src/Recruit.Storage.Client/Core/Repositories/MongoDbVacancyRepository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Esfa.Recruit.Storage.Client.Core.Entities;
+using Esfa.Recruit.Storage.Client.Core.Entities.VacancyPatches;
+using Esfa.Recruit.Storage.Client.Core.Mongo;
+using Microsoft.Extensions.Options;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using System;
+using System.Threading.Tasks;
+
+namespace Esfa.Recruit.Storage.Client.Core.Repositories
+{
+    public class MongoDbVacancyRepository : MongoDbCollectionBase, ICommandVacancyRepository, IQueryVacancyRepository
+    {
+
+        private const string Database = "Recruit";
+        private const string Collection = "Vacancy";
```

Comment:
I think it is common to use the plural for collection names and lowercase. http://arkusnexus.com/2016/09/12/coding-guidelines-mongodb/?nabe=5802299602763776:0

Reply:
👍